docs/api/scheduler.md

```diff
@@ -337,7 +337,7 @@ The content of the application object is the same as Queue Applications. See
 
 ### Queue applications
 
-Fetch all Applications for the given Partition/Queue combination and displays general information about the applications like used resources, queue name, submission time and allocations.
+Fetch all Applications for the given Partition/Queue combination and displays general information about the applications like used resources, queue name, submission time and allocations. In case queue name contains any special characters, needs to be url escaped to avoid issues.
```

Comment:
Nit: "In case the queue name contains any special characters, it needs to be url escaped to avoid issues."